src/sentry/models/relocation.py

```diff
@@ -81,7 +81,7 @@ def get_choices(cls) -> list[tuple[int, str]]:
     # This is left NULL because we want to retain the ability to audit and rollback a `Relocation`
     # even in the event of the owner's account being deleted, but that `NULL` should never be set at
     # creation time.
-    owner = HybridCloudForeignKey("sentry.User", null=True, on_delete="SET_NULL")
+    owner_id = BoundedBigIntegerField(null=True)
```

Comment:
Do you plan to retain the behaviour of setting `creator_id` and/or `owner_id` to be `null` when the associated `User` is deleted?

Reply:
I think it may be a useful addition in the future, but I've gone ahead and removed it for now.